### PR TITLE
Update the network sync status detection.

### PR DESCRIPTION
**Pull request by geek96**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +21,22 @@
...
     "prompt": "1.0.0",
+    "redis": "2.8.0",
     "require-injector": "3.1.6",
-    "sequelize": "4.37.6",
+    "sequelize": "4.37.7",
+    "sha256": "0.2.0",
     "shutdown-handler": "1.0.1",
     "sql2csv": "1.3.0",
-    "web3": "1.0.0-beta.34"
+    "web3": "github:dskvr/web3.js#1.0"
```
Comment (DanailMinchev) [inline package.json]: Why we need forked `dskvr/web3.js` here? Thanks

Comment (geek96) [reply package.json]: I am not sure , its added by @sandw1ch and he fixed some something in `dskvr/web3.js`.

Comment (DanailMinchev) [reply package.json]: Thanks for clarification :+1:

**Comment by dskvr** [comment]
Testing this soon.

File: package.json (hunk 1 above)
Comment (dskvr) [reply package.json]: @DanailMinchev That change is in #37

**Comment by dskvr** [comment]
@geek96 This merge is difficult to follow because it includes merge from optimizations. Instead, could you please PR the optimizations branch with this change only?

**Comment by dskvr** [comment]
I cannot accept this merge into master

**Pull request closed without merging**